packages/samples/headless-react/src/components/facet-manager/facet-manager.class.tsx

```diff
@@ -46,6 +46,10 @@ export class FacetManager extends Component<{
   }
 
   render() {
+    if (!this.state) {
+      return this.props.children;
```

Comment:
The facet manager is the only component that doesn't return `null` when its state doesn't exist. It returns its children in order for their `componentDidMount` to be executed before `SearchPage`'s, which ensures that search parameters are initialized after facets.